_attributes/common-attributes.adoc

```diff
@@ -183,4 +183,4 @@ endif::[]
 :web-terminal-op: Web Terminal Operator
 :devworkspace-op: DevWorkspace Operator
 
-:ocp-product-title: OpenShift Container Platform
\ No newline at end of file
+:ocp-product-title: OpenShift Container Platform
```

Comment:
This is generally already defined: https://github.com/openshift/openshift-docs/pull/63401/files#diff-8f8ad76b1fd362614680cd9cd3719aa7d275414cd3c3e5e52b00bb3b6e5de67bR1-R4
Not sure if that mechanism works on the `build-docs` branch though. Also, if you are creating your own version of the common attributes file on this branch, it might be better to start with a clean file, since most of what's in this copied version probably won't apply for the Builds doc set, e.g. all the other product names and things.

Reply:
The mechanism did not work. This is why added a new attribute :ocp-product-title: in the attributes file.
I also removed the unnecessary attributes from the file.